### DISCUSSION TITLE
Tables created in the wrong database

### DISCUSSION THREAD

**Discussion by dekiesel** [pgmodeler/pgmodeler#1850] [Q&A] [answered]
Hi,

I am trying to apply the sql-export of my model.

It starts with a few `CREATE ROLE`s, followed by `CREATE DATABASE mytestdb`. That part works as expected.

But the following `CREATE TABLE`s are all applied in the `postgres` database, not in `mytestdb`.

How should I set up my project so that when I apply a `.sql` it creates the tables in the recently created database?

**Comment by dekiesel** [comment]
![image](https://github.com/pgmodeler/pgmodeler/assets/17333827/921e8fc8-31e1-4b0b-b761-60c3cabe3058)

Atm I am using `\c mytestdb` under Custom SQL. Is that the recommended way of doing it?

**Comment by rkhaotix** [comment, marked as answer]
Hi @dekiesel 

The .sql file is created in the way pgModeler runs the commands internally. Having said that, if you want to run the script as-is and create the objects correctly, use the export tool either in GUI or CLI (pgmodeler-cli). 

If you want to run the script manually (for example, via ```psql```) , you have to edit the file and put a ```\c [database]``` after the line that creates and sets the permissions over the new database so the PostgreSQL client can connect to the correct database and run the rest of commands. The way you are doing is another alternative to workaround this issue. Anyway, I can try to add an instruction in the generated file so you don't need to keep adding the ```\c``` every time.

**Reply by dekiesel** [reply]
That'd be great, thank you!
